configure.ac

```diff
@@ -119,6 +121,24 @@ AS_IF([test "x$enable_libunibreak" != xno], [
     ])
 ])
 
+AS_IF([test "x$enable_pthreads" = xyes && test "x$ac_cv_header_stdatomic_h" != xyes], [
```

Comment:
Nit: currently the pthread check is placed in the middle of a block of dependency checks, and because/when `AX_PTHREAD` runs multiple checks internally, it breaks up the block in the output, making it harder for a human to read:

```
checking for pkg-config... /mingw32/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for FREETYPE... yes
checking for FRIBIDI... yes
checking for HARFBUZZ... yes
checking for LIBPNG... yes
checking how to run the C preprocessor... gcc -mincoming-stack-boundary=2 -fsanitize=undefined -fsanitize=float-cast-overflow -fsanitize-undefined-trap-on-error -fno-sanitize-recover=all -E
checking whether gcc -mincoming-stack-boundary=2 -fsanitize=undefined -fsanitize=float-cast-overflow -fsanitize-undefined-trap-on-error -fno-sanitize-recover=all is Clang... no
checking whether pthreads work with "-pthread" and "-lpthread"... yes
checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE
checking whether more special flags are required for pthreads... no
checking for PTHREAD_PRIO_INHERIT... yes
checking for FONTCONFIG... yes
checking for CORETEXT... no
checking for DIRECTWRITE... yes
checking for Win32 desktop APIs... yes
checking for nasm... yes
checking if nasm supports vpmovzxwd... yes
checking for git... no
```

I think it would be more convenient to move it just before the FreeType check.